libvast/native-plugins/feather.cpp

```diff
@@ -24,6 +25,23 @@
 
 namespace vast::plugins::feather {
 
+/// Configuration for the Feather plugin.
+struct configuration {
+  int64_t zstd_compression_level{9};
+
+  template <class Inspector>
+  friend auto inspect(Inspector& f, configuration& x) {
+    return f(x.zstd_compression_level);
+  }
+
+  static const record_type& layout() noexcept {
+    static auto result = record_type{
+      {"zstd-compression-level", integer_type{}},
```

Comment:
This is `vast.plugins.feather.zstd-compression-level`, right? I think this should be `vast.compression-level` instead and apply to both plugins at the same time. That'd be a lot easier to communicate in documentation.

Reply:
That makes sense. I'm not aware of a way to access the overall system config from within a plugin, though.

Reply:
For any given actor that runs in the server process you can call `content(self->system().config())` to get the configuration as a `caf::settings` object. This may need to be passed in from the `store_actor_plugin` to the `store_plugin`.

Reply:
Would it make sense to also adapt `chunk::compress` and `arrow_table_slice_builder::create_table_slice` to use this new system-wide default? I'm a bit hesitant b/c tradeoffs for storage vs other data representations may be different / separate concerns.

Reply:
I think those are implementation details and should not be touched in the context of this PR.

Reply:
I think it actually makes sense to have a `zstd-compression-level`. A compression level without a specific algorithm is meaningless. The range is different, some go from 1 to 3, some to 15, etc.

Reply:
Agree on naming it `vast.zstd-compression-level` rather than just `vast.compression-level`, but I still think it shouldn't be `vast.plugins.{parquet,feather}.zstd-compression-level`, as that's just very hidden.

Reply:
Agreed as well.